products/views.py

```diff
@@ -1,3 +1,31 @@
-from django.shortcuts import render
+from django.http        import JsonResponse
+from django.views       import View
 
-# Create your views here.
+from products.models    import Product, ThumbnailImage, DetailImage
+from categories.models  import FirstCategory, SecondCategory
+
+class ProductItemView(View):
+    def get(self, request, product_id):
+        try:
+            product = Product.objects.get(id=product_id)
+            second_category = SecondCategory.objects.get(id=product.second_category.id)
+
+            result = {
+                'title'           : product.title,
+                'description'     : product.description,
+                'first_category'  : second_category.first_category.title,
+                'second_category' : second_category.title,
```

Comment:
```suggestion
                'first_category'  : product.second_category.first_category.title,
                'second_category' : product.second_category.title,
```
위에서 second_category를 변수 지정할 필요없이 이렇게 사용 가능할 것 같아요!

Reply:
수정했습니다!